blog/_posts/2019-10-07-vs-code-latex-extension.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Fortgeschrittene vscode-Extension für LaTeX
+author: Ludwig Neste
+---
+In der Installationsanleitung wird die LaTeX-Extension `LaTeX language support` installiert.
+Diese kann Syntax Highlighting (Passende Einfärbung des LaTeX-Textes, für die Übersichtlichkeit), Snippets (Kleinere LaTeX-Code-Auszüge, die man häufig verwendet, sodass man diese nicht andauernd tippen muss, sondern z.B. mit `str`+`leertaste` snippets auswählen kann) und folding (z.B. Verstecken einer Mathe-Umgebung, für die Übersichtlichkeit).
+
+Es gibt allerdings eine noch viel mächtigere Extension, die euch die Arbeit mit LaTeX in vscode erleichtert. Diese heißt `LaTex Workshop` von James Yu. Dieses kann alles, was auch `LaTeX language support` kann, jedoch zusätlich noch: PDFs anzeigen, code-completion mit intellisense 
+(zeigt dir an, was du mit deinem angefangenen code meinen könntest, z.B.: tippe `\autoref{fig:` -> zeigt dir alle Referenz mit `fig:` beginnend an.) und es zeigt dir sogar eine Vorschau der Mathe-Sachen die du gerade tippst (FYI: mit [MathJax](https://www.mathjax.org/)). 
+Das ganze Add-On kann noch vieles mehr, und wer interessiert ist, sollte sich definitiv mit dem ganzen genauer beschäftigen (Stichworte [SynchTex](https://github.com/James-Yu/LaTeX-Workshop/wiki/View) und [Linting](https://github.com/James-Yu/LaTeX-Workshop/wiki/Linters)).
+
+Da wir jedoch immernoch die im Toolbox-Workshop gelernte Toolchain benutzen wollen, müssen wir die Extension daran hindern, dass sie die LaTeX-Dokumente selber baut. (Es ist auch möglich, die Extension so zu konfigurieren, dass sie die genau so baut, wie ihr das auch machen würdet, immer wenn ihr abspeichert. Das müsst ihr aber selbst herausfinden. &#x1f61c;)
```

Comment:
How about unicode?

Reply:
I like to write what's directly on my keyboard... but ok...
```suggestion
Da wir jedoch immernoch die im Toolbox-Workshop gelernte Toolchain benutzen wollen, müssen wir die Extension daran hindern, dass sie die LaTeX-Dokumente selber baut. (Es ist auch möglich, die Extension so zu konfigurieren, dass sie die genau so baut, wie ihr das auch machen würdet, immer wenn ihr abspeichert. Das müsst ihr aber selbst herausfinden. 😜)
```

Reply:
Code is more often read than written.  Now I can see it's a nice smiley, before I had to google the escape sequence.